### ISSUE TITLE
Bug: backend.js Uncaught TypeError: Cannot read property 'sub' of undefined

### ISSUE THREAD

**Issue by steida** [facebook/react#18205]
```
backend.js:32 Uncaught TypeError: Cannot read property 'sub' of undefined
    at g (backend.js:32)
    at e (backend.js:8)
```

It happens when a server returns a string without HTML. I suppose React dev tools expect DOM.

![Screenshot 2020-03-03 at 21 43 42](https://user-images.githubusercontent.com/66249/75817818-18c1fd00-5d98-11ea-89f5-547e97c44632.png)

**Comment by bvaughn** [comment]
DevTools only injects if the document `contentType` is `text/html` (see #17739). This change also required adding some checks for "'sub' of undefined" (see #17848).

Unfortunately the first change was released in v4.4.0 (January 3, 2020) and the second change has not yet been released. So this is a known issue and will be fixed by v4.4.1/v4.5.0 whenever it's released...which should be soon. I guess probably this week.

Sorry for the inconvenience.

**Comment by bvaughn** [comment]
Fixed in v4.5

https://github.com/facebook/react/blob/master/packages/react-devtools/CHANGELOG.md#450-march-3-2020
